src/lwpCall.js

```diff
@@ -117,6 +117,24 @@ export default class {
     }
   }
 
+  remoteDisplayName() {

```

Comment:
if you're returning initials only, then you should express that in the function name as `remoteDisplayName` is mealing

Reply:
Renamed function to **remoteNameInitials**

Reply:
i don't agree in adding these kind of functions (very specific) in the core libwebphone. i would create `remoteDisplayName` if its not there, the use that function to create any derivatives in the client using the core libwebphone. but @k-anderson would know better